rubyric/02_getting_started.rst

```diff
@@ -24,6 +24,22 @@ a new menu item in A+ choose Edit course and Menu and press Add new menu item.
 You can also choose to use Rubyric through Haka login. In that case you don't
 need to add Rubyric to menu.
 
+.. important::
+
+  If you want to test Rubyric locally, you must read the
+  `Official Rubyric documentation <https://github.com/Aalto-LeTech/rubyric/blob/master/doc/rubyric.md#connect-local-rubyric-to-a-course-in-docker-optional>`_
+  and follow the instructions on how to use a local copy of Rubyric along with A+.
+  It is also important to mention that currently testing Rubyric locally can be
+  slightly difficult. Therefore, we recommend adding the desired `submit`
+  directives to the course along with the corresponding `config.yaml` file,
+  which should contains the configuration of the Rubyric exercise. Once you have
+  verified manually that all the parameters are correct, you can deploy the course
+  in our `testing environment <https://minus.cs.aalto.fi>`_ and only then verify that
```

Comment:
What if you are from Tampere or Chosun? Should this be written as "Then you can push to your testing server (e.g. minus.cs in Aalto CS dep)"?

Reply:
I actually do not know what Chosun is?. But yes, it makes sense to write the sentence in the way you suggest, since the minus.cs server is basically hosted at Aalto and mainly for Aalto people.

Reply:
> I actually do not know what Chosun is?.

Heh. I thought it was the towns name, but it was actually name of the University in the town. Any case, in this context, it's one location where A+ is used.

Reply:
Minus.cs is only used at Aalto. If the reader is not Aalto, then they just test in their own A+ server. Jaakko's wording is appropriate, but "department" is abbreviated "dept.", not "dep".